go/smoke-test/action.yml

```diff
@@ -36,6 +51,12 @@ runs:
       run: |
         go test ./tests/smoke --tags=smoke
       shell: bash
+    - name: Rollback on failure
+      if: failure() $$ inputs.ROLLBACK_CLUSTER != null
+      run: |
+        aws eks update-kubeconfig --name ${{ inputs.ROLLBACK_CLUSTER }} --role-arn ${{ inputs.ROLLBACK_ROLE }} --region ${{ inputs.ROLLBACK_AWS_REGION }}
+        kubectl rollout undo deployment/${{ inputs.ROLLBACK_DEPLOYMENT }} -n ${{ inputs.ROLLBACK_NAMESPACE }}
+      shell: bash
```

Comment:
Let's use Dan's work on this where we can 😄 : https://github.com/wishabi/curator/blob/main/.github/workflows/CI.yml#L181

Reply:
Oh, that's much nicer! I grabbed what was in `distribution-api` as I thought that was our "golden repo".  I think this is because this won't work for SRMS that it's not there. Updated.